client/Configurator.c

```diff
@@ -216,12 +216,19 @@ static void udpInterface(Dict* config, struct Context* ctx)
                 key = String_clone(key, perCallAlloc);
                 char* lastColon = CString_strrchr(key->bytes, ':');
 
-                if (!Sockaddr_parse(key->bytes, NULL)) {
-                    // it's a sockaddr, fall through
-                } else if (lastColon) {
-                    // try it as a hostname.
-                    Log_critical(ctx->logger, "Couldn't add connection [%s], "
-                                                "hostnames aren't supported.", key->bytes);
+                if (lastColon) {
+                    if (!Sockaddr_parse(key->bytes, NULL)) {
```

Comment:
I digged into the code and you're right, it's actually expecting a string here, sorry.